nise/__main__.py

```diff
@@ -526,7 +533,7 @@ def main():
     options = vars(args)
     _, provider_type = _validate_provider_inputs(parser, options)
 
-    if not options.get('start_date'):
+    if not (options.get('start_date') or options.get('static_report_file')):
```

Comment:
Call me crazy, but this just looks off. If either or both are passed, then no error. But doesn't that allow the condition for static file w/o start date? Maybe it's too early in the morning.

Reply:
There is a start date check in the `run()` function now to catch static files without a `start_date`.